src/commands/performance/bandwidth_throttle/show.ts

```diff
@@ -0,0 +1,12 @@
+import { runCommand } from '../../../util/command';
+import { writeConfiguration } from './util';
+
+export = {
+	command: 'show',
+	description: 'Display current bandwidth throttle configuration',
+	handler: argv => {
+		runCommand(argv, async (client, output) => {
+			writeConfiguration(await client.performance.get(), output);
+		});
+	}
```

Comment:
If I've never configured any throttling and run the show command, what should I see? Are there default values or uninitialized values that should be returned? Right now I see no values returned, or empty json (no properties) when I use --json.